src/app/routes/clientRoutes.js

```diff
@@ -161,6 +161,8 @@ const renderDefaultIndexPage = (req, res) => {
   const mobileDetect = new MobileDetect(req.headers['user-agent'])
   if ((req.path == '/get' || req.path == `/${req.params.slug}/get`) && mobileDetect.os() == 'AndroidOS') {
     res.redirect(envHelper.ANDROID_APP_URL)
+  } else if (req.path.includes('/play/quiz')) {
```

Comment:
Write separate method for play route. like
app.all(['play/*'], playContent(false));

const playContent = (req, res) => {
if('play/quiz'){
  // Redirect to quiz index.html page
} else {
  // redirect to renderDefaultIndexPage
}
}